README.md

```diff
@@ -29,9 +30,17 @@ In overview, your file structure should be as follows.
     ├── frontend
     |   └── Dockerfile
     └── backend
-        └── Dockerfile
+        ├── Dockerfile
+        └── aethel_db
+            └── data
+                └── aethel.pickle
```

Comment:
I don't like that users have to create this file structure when they start this project. Ideally, I would have put `aethel.pickle` in the root directory or in `backend`, but I did not manage to do this in `backend/Dockerfile`. If you know how to fix this, do let me know!

Reply:
No, I was just trying to keep the data files (both the full set and the subset) in the same location, but I guess it's not too bad if the full dataset is in the root directory while the subset is put somewhere deep inside `aethel_db/data/` (especially since you'll only need it for dev).